segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/rolling/RollingStorage.java

```diff
@@ -40,6 +42,13 @@
 import lombok.extern.slf4j.Slf4j;
 import lombok.val;
 
+import static io.pravega.common.tracing.RequestTracker.buildRequestDescriptor;
```

Comment:
Please see my comments in `StreamSegmentContainer`. I really do not believe we should be going down to this level with request tracing. You are essentially adding another coupling from this to the RequestTracker (and implicity to Guava Cache) from this class, which make it harder to read and harder to change in the future.

Reply:
This has been removed along with all the code related to track Tier-2 activity.